### ISSUE TITLE
Cannot run example model 'sir' after latest update

### ISSUE THREAD

**Issue by thomas-papaioannou** [climateinteractive/SDEverywhere#488]
Hello.

npm create '@sdeverywhere@latest' is working fine, but when running 'npm run dev'

I get the following error. Note: Previous week it was working just fine. I believe it due to the latest update in main.

```
> project@1.0.0 dev
> sde dev

Generating files...
Done generating files (0.0s)
Generating model...
Done generating model (1.4s)
Building worker
[Error: Could not load C:\Users\thomas.papaioannou\Documents\SDEverywhere\models\sir\sde-prep\staged\model\wasm-model.js (imported by node_modules/@sdeverywhere/plugin-worker/template-worker/worker.js): ENOENT: no such file or directory, open 'C:\Users\thomas.papaioannou\Documents\SDEverywhere\models\sir\sde-prep\staged\model\wasm-model.js'] {
  errno: -4058,
  code: 'PLUGIN_ERROR',
  syscall: 'open',
  path: 'C:\\Users\\thomas.papaioannou\\Documents\\SDEverywhere\\models\\sir\\sde-prep\\staged\\model\\wasm-model.js',
  pluginCode: 'ENOENT',
  plugin: 'vite:load-fallback',
  hook: 'load',
  watchFiles: [
    'C:/Users/thomas.papaioannou/Documents/SDEverywhere/models/sir/node_modules/@sdeverywhere/plugin-worker/template-worker/worker.js',
    'C:/Users/thomas.papaioannou/Documents/SDEverywhere/models/sir/node_modules/@sdeverywhere/runtime/dist/index.js',
    'C:/Users/thomas.papaioannou/Documents/SDEverywhere/models/sir/node_modules/@sdeverywhere/runtime-async/dist/worker.js',
    '\x00virtual:model-spec',
    'C:/Users/thomas.papaioannou/Documents/SDEverywhere/models/sir/node_modules/threads/worker.mjs',
    'C:\\Users\\thomas.papaioannou\\Documents\\SDEverywhere\\models\\sir\\sde-prep\\staged\\model\\wasm-model.js'
  ]
}
```

**Comment by chrispcampbell** [comment]
Hi @thomas-papaioannou, thank you for filing this.

The issue is that the `@sdeverywhere/create` package downloads the template files from the `examples` directory for the `main` branch.  Normally this is what we want, and those latest template files work in concert with the latest packages published in the npm registry.  However, these past two weeks we've been in an unusual situation where I'm merging a few big branches and holding back on npm publishing them until the dust settles, since there are some small breaking changes involved.  This means that the template files are now "ahead" of the published packages, which led to the mismatch that caused the error above.

TL;DR, I will push a temporary fix to the `sde.config.js` files in the templates now so that they generate C code (the old default behavior).  Since you probably already have the project set up locally from your last attempt, you can simply open `sde.config.js` and change `const genFormat = 'js'` to `const genFormat = 'c'` and then retry the `npm run dev` and it should work.  Or, once I push the fix in a few minutes, you can try the `npm create @sdeverywhere@latest` to go through the full setup process again and that should work too.
